README.md

```diff
@@ -39,12 +32,45 @@ def application do
 end
 ```
 
+## If you'd like to mount VerkWeb on another Endpoint:
+
+
+```elixir
+defmodule MyApp.Endpoint do
+  use VerkWeb.Mount, path: "/verk"
+  ...
+end
+```
+
+That should be it! :)
+
+## If you'd like to run VerkWeb as stand-alone Endpoint on a different port than the main application:
```

Comment:
I don't quite understand the question... In lines 51 to 53 -> you set the port to whatever you wish and you turn the server on. Is that enough to point that "you can configure the port"?